client/src/pages/RoomPage/index.tsx

```diff
@@ -191,8 +192,15 @@ function RoomPage({ name }: { name: string }): JSX.Element {
   }, []);
 
   useEffect(() => {
-    globalUserInfos = userInfos;
-  }, [userInfos]);
+    if (stream) {
```

Comment:
Seems unrelated? A bit confused about this

Reply:
Seems like a residue from when I was merging main into this